src/accelerate/test_utils/testing.py

```diff
@@ -389,6 +389,24 @@ def tearDown(self):
         PartialState._reset_state()
 
 
+class LaunchTestCase(unittest.TestCase):
+    """
+    A TestCase class that helps prepare and run a test with `accelerate.launch`.
+    """
+
+    def get_launch_command(self, **kwargs):
+        command = ["accelerate", "launch"]
+        for k, v in kwargs.items():
+            if v is not None:
+                command.append(f"--{k}={v}")
+            elif isinstance(v, bool) and v:
+                command.append(f"--{k}")
+        return command
+
+    def setUp(self):
+        self.default_command = self.get_launch_command(num_processes=device_count)
```

Comment:
The idea here is that it's not a function, this is what 99% of all the classes use/repeat the logic of. Calling the functional `get_launch_command` is only done in a few areas otherwise.

Reply:
It could be a constant too.